janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -366,7 +366,6 @@ public void mutate(final StaticBuffer key, final List<Entry> additions, final Li
 
     @Override
     public void acquireLock(final StaticBuffer key, final StaticBuffer column, final StaticBuffer expectedValue, final StoreTransaction txh) throws BackendException {
-        throw new UnsupportedOperationException();
```

Comment:
Rather than remove this can you you wrap it in a conditional statement based off of `EXTERNAL_LOCKING`

Reply:
Done